docs/examples/tcloud_getting_started.md

```diff
@@ -0,0 +1,230 @@
+# Tobiko Cloud: Getting Started
```

Comment:
Why is this in `examples/`?

Reply:
"Note: will update the nav bar configs once @mesmith027 merges in the latest nav bar structure for tcloud docs."

Reply:
ah, sorry, missed that